Formula/open-ocd.rb

```diff
@@ -7,6 +7,7 @@ class OpenOcd < Formula
 
   livecheck do
     url :stable
+    regex(%r{url=.*?/openocd[._-]v?(\d+(?:\.\d+)+(?:[a-z]\d+)?)\.t}i)
```

Comment:
```suggestion
    regex(%r{url=.*?/openocd[._-]v?(\d+(?:\.\d+)+)\.t}i)
```

I don't see any versions with a trailing letter and number after the numeric version, so I think we can safely remove it. This brings it in line with the most common regex format for `livecheck` blocks that use the `Sourceforge` strategy (which is simply a modification of the standard regex for matching versions from filenames in `href` attributes).

Reply:
that is right, copy paste error :(